src/tag/styled-components.js

```diff
@@ -77,7 +81,7 @@ export const Root = styled('span', props => {
     color: color,
     opacity: $disabled ? '.56' : null,
     borderRadius: $theme.borders.useRoundedCorners
-      ? $theme.borders.radius100
+      ? $theme.borders.radius400
```

Comment:
Could you also add to it
```    
paddingLeft: '12px',
boxSizing: 'border-box',
```
And reduce padding on the `Action` element to make it fit into the container.
The `delete` icon's size could also be adjusted to `10px`.
So with the above it will be close to the Figma design:
<img width="136" alt="screen shot 2018-11-07 at 12 11 47 pm" src="https://user-images.githubusercontent.com/1920399/48158087-643cc780-e286-11e8-81d6-47c5bae2643c.png">

Reply:
I would also change `Action`'s element type to `span` since the `Root` is an inline element.

Reply:
@nadiia updated - can you take a look?

Reply:
Looks great, thanks!